plugins/woocommerce/includes/admin/meta-boxes/class-wc-meta-box-order-data.php

```diff
@@ -214,70 +214,86 @@ public static function output( $post ) {
 			<input name="post_title" type="hidden" value="<?php echo esc_attr( empty( $order->get_title() ) ? __( 'Order', 'woocommerce' ) : $order->get_title() ); ?>" />
 			<input name="post_status" type="hidden" value="<?php echo esc_attr( $order->get_status() ); ?>" />
 			<div id="order_data" class="panel woocommerce-order-data">
-				<h2 class="woocommerce-order-data__heading">
-					<?php
-
-					printf(
-						/* translators: 1: order type 2: order number */
-						esc_html__( '%1$s #%2$s details', 'woocommerce' ),
-						esc_html( $order_type_object->labels->singular_name ),
-						esc_html( $order->get_order_number() )
-					);
-
-					?>
-				</h2>
-				<p class="woocommerce-order-data__meta order_number">
-					<?php
-
-					$meta_list = array();
-
-					if ( $payment_method && 'other' !== $payment_method ) {
-						$payment_method_string = sprintf(
-							/* translators: %s: payment method */
-							__( 'Payment via %s', 'woocommerce' ),
-							esc_html( isset( $payment_gateways[ $payment_method ] ) ? $payment_gateways[ $payment_method ]->get_title() : $payment_method )
-						);
-
-						$transaction_id = $order->get_transaction_id();
-						if ( $transaction_id ) {
-
-							$to_add = null;
-							if ( isset( $payment_gateways[ $payment_method ] ) ) {
-								$url = $payment_gateways[ $payment_method ]->get_transaction_url( $order );
-								if ( $url ) {
-									$to_add .= ' (<a href="' . esc_url( $url ) . '" target="_blank">' . esc_html( $transaction_id ) . '</a>)';
+				<div class="order_data_header">
```

Comment:
If you're not using "Hide whitespace", you can activate it to see the changes clearer.